src/runtime/lpython_builtin.py

```diff
@@ -1051,16 +1049,42 @@ def _lpython_str_isascii(s: str) -> bool:
             return False
     return True
 
-def _lpython_str_isspace(s:str) -> bool:
+def _lpython_str_isspace(s: str) -> bool:
+    # A Unicode character is considered a 'whitespace' if it has has a bidirectional
+    # type 'WS', 'B' or 'S'; or the category 'Zs'.
     if len(s) == 0:
         return False
-    ch: str 
-    for ch in s:
-        if ch != ' ' and ch != '\t' and ch != '\n' and ch != '\r' and ch != '\f' and ch != '\v':
+    
+    char: str
```

Comment:
Sometimes `char` keyword is not mangled in C backend, so it fails, rename it to `ch` or `c` or something

Reply:
Oh! :smile: